asar-teensy-firmware/TestScripts/XBEE/XBEE_classTest/XBEE.cpp

```diff
@@ -0,0 +1,133 @@
+/* XBEE.cpp
+ *  Author: Aaron Mallabar
+ *  Date:   March 6,2018
+ *  Description: Header file that wraps us the serial commands to read info from the XBEE
+ *				and interprets the information into executable directions
+ *  
+ */
+
+#include "Arduino.h"
+#include "XBEE.h"
+#include "Wire.h"
+
+namespace ASAR
+{
+	XBEE::XBEE()
+	{
+		Serial1.begin(115200);      	//Initialize Xbee, with baud rate
+	}
+
+	XBEE::~XBEE()
+	{
+		//empty destructor
+	}
+
+	/*Read a single Instruction from xbee*/
+	void XBEE::readRawInstruct()
+	{
+	  if (Serial1.available() && Serial1.read() == 0xFF)               //Only if there is something from xbee to read
+	  {
+	    for (int i = 0; i <= 5; i++)        //Read all the inputs (until stop bit is read)
+	   	{
+				rawRead[i] = Serial1.read();             //and fill up the temporary array
+	      	//Serial.print("Argument # "); Serial.print(i);
+	      	//Serial.print(", Raw reading: "); Serial.println(rawRead[i], HEX); 
+	      if(rawRead[i] == 0xF0)
+	      {
+	       	i = 6;
+	  		}
+	      else if (i >= 4)
+	      {
+	        Serial.println("ERROR-001: Stop Bit Not Detected");
+	      }
+	   	}
+  	}                                   
+	}
+
+
+  /*Interprets a single raw reading instruction, fills relevant arrays 
+   * with the index being the number of instruction */
+  void XBEE::Interpret_instruct()
```

Comment:
Inconsistent tabs inside the namespace.

Reply:
Either this changed between then and now, or I don't know what you are talking about